go.mod

```diff
@@ -1,3 +1,3 @@
-module github.com/mailru/go-clickhouse
```

Comment:
please revert this change

Reply:
ouch, it wasn't supposed to be here, apologies